index.yaml

```diff
@@ -3957,6 +3957,21 @@ entries:
     - assets/rancher-vsphere-csi/rancher-vsphere-csi-2.1.000.tgz
     version: 2.1.000
   rancher-webhook:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-webhook
+    apiVersion: v2
+    appVersion: 0.1.2-rc1
+    created: "2021-09-01T14:12:37.396066-07:00"
+    description: ValidatingAdmissionWebhook for Rancher types
+    digest: 104e663b2a189f5de0afd3c93f04e4b7ee698420100f48924204697beac83df7
+    name: rancher-webhook
+    urls:
+    - assets/rancher-webhook/rancher-webhook-0.1.200-rc1+up0.1.2-rc1.tgz
+    version: 0.1.200-rc1+up0.1.2-rc1
```

Comment:
note: The version will be `0.1.200+up0.1.2` when the upstream chart `0.1.2` is used.